main_2024.py

```diff
@@ -111,6 +112,10 @@ def main() -> int:
         GEOLOCATION_CAMERA_ORIENTATION_YAW = config["geolocation"]["camera_orientation_yaw"]
         GEOLOCATION_CAMERA_ORIENTATION_PITCH = config["geolocation"]["camera_orientation_pitch"]
         GEOLOCATION_CAMERA_ORIENTATION_ROLL = config["geolocation"]["camera_orientation_roll"]
+
+        MIN_ACTIVATION_THRESHOLD = config["cluster_merge"]["min_activation_threshold"]
+        MIN_NEW_POINTS_TO_RUN = config["cluster_merge"]["min_new_points_to_run"]
+        RANDOM_STATE = config["cluster_merge"]["random_state"]
```

Comment:
I think you forgot to change the name from `cluster_merge` to `cluster_estimation` here. Can you try running main as well so that we don't miss errors like these as well. If you need help with that ping me and let me know.